packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CirclesController.java

```diff
@@ -67,7 +67,15 @@ boolean onCircleTap(String googleCircleId) {
     if (circleId == null) {
       return false;
     }
-    methodChannel.invokeMethod("circle#onTap", Convert.circleIdToJson(circleId));
+    flutterApi.onCircleTap(
+        circleId,
+        new Messages.VoidResult() {
+          @Override
+          public void success() {}
+
+          @Override
+          public void error(@NonNull Throwable error) {}
```

Comment:
The main reason I didn't add it in this PR is that I know we have a class of issues around lifetime mismatch between native and Dart in Google Maps, and I worry that just adding logging here without deeply investigating those first would cause a bunch of spam where the call fails only because the instance has been eliminated on the Dart side, in which case silently dropping them is better than logging.